package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha && eslint .",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "storybook": "(cd node_modules/apostrophe-storybook && start-storybook -p 6006)",
+    "build-storybook": "build-storybook"
```

Comment:
How does this one work? Is build-storybook globally installed or something?

Reply:
oops this is a stump .. this is what we need to write, will remove